diff --git a/node_modules/frames.js/dist/chunk-CMKMOD2M.js b/node_modules/frames.js/dist/chunk-CMKMOD2M.js
index 5b97730..95380ee 100644
--- a/node_modules/frames.js/dist/chunk-CMKMOD2M.js
+++ b/node_modules/frames.js/dist/chunk-CMKMOD2M.js
@@ -1,7 +1,7 @@
 // src/next/getCurrentUrl.ts
 function getCurrentUrl(req) {
   const scheme = process.env.NODE_ENV === "production" ? "https://" : "http://";
-  const appUrl = process.env.VERCEL_URL || process.env.APP_URL;
+  const appUrl = process.env.APP_URL || process.env.VERCEL_URL;
   const host = req.headers?.host;
   const pathname = req.url?.startsWith("/") ? req.url : req.url ? new URL(req.url).pathname + new URL(req.url).search : "";
   const parsedAppUrl = appUrl ? appUrl.startsWith("http://") || appUrl.startsWith("https://") ? new URL(pathname, appUrl) : new URL(pathname, scheme + appUrl) : void 0;
diff --git a/node_modules/frames.js/dist/next/getCurrentUrl.cjs b/node_modules/frames.js/dist/next/getCurrentUrl.cjs
index 23bea13..68edd0f 100644
--- a/node_modules/frames.js/dist/next/getCurrentUrl.cjs
+++ b/node_modules/frames.js/dist/next/getCurrentUrl.cjs
@@ -25,7 +25,7 @@ __export(getCurrentUrl_exports, {
 module.exports = __toCommonJS(getCurrentUrl_exports);
 function getCurrentUrl(req) {
   const scheme = process.env.NODE_ENV === "production" ? "https://" : "http://";
-  const appUrl = process.env.VERCEL_URL || process.env.APP_URL;
+  const appUrl = process.env.APP_URL || process.env.VERCEL_URL;
   const host = req.headers?.host;
   const pathname = req.url?.startsWith("/") ? req.url : req.url ? new URL(req.url).pathname + new URL(req.url).search : "";
   const parsedAppUrl = appUrl ? appUrl.startsWith("http://") || appUrl.startsWith("https://") ? new URL(pathname, appUrl) : new URL(pathname, scheme + appUrl) : void 0;
